toolchest_client/_internal_utils.py

```diff
@@ -9,7 +9,7 @@
 def _validate_tool_kwargs(**kwargs):
     """Validates kwargs passed to (specific) tool functions."""
 
-    # Defaults for query parameters input_name and output_name are provided
+    # Default values for query parameters input_name and output_name are provided
     # by default in tool function; they should not be provided by the user.
```

Comment:
What do you mean here by "they should not be provided by the user?"

It seems reasonable that someone would run this like:
```
>>> toolchest.cutadapt(
... tool_args="",
... input_path="./temp/short.fastq",
... output_path="./output.fastq"
... )
```

Reply:
@bcai2 can you still respond to this?